ha/const.py

```diff
@@ -318,3 +321,8 @@ class CLUSTER_STATUS(Enum):
     DEGRADED = "degraded"
     UNHEALTHY = "unhealthy"
     ONLINE = "online"
+
+class BMC_POWER_STATUS(Enum, metaclass=EnumListMeta):
```

Comment:
BMC_POWER_STATUS

can we change to 

POWER_STATUS

here i think BMC should be abstract

Reply:
Changed to SERVER_POWER_STATUS